src/lib/checkbox/checkbox.scss

```diff
@@ -192,6 +192,7 @@ $md-checkbox-ripple-size: 15px;
 }
 
 md-checkbox {
+  margin: 16px 0;
```

Comment:
We can't do this- none of the components can define a margin on their outer element.